fastly/ftp.go

```diff
@@ -15,11 +15,12 @@ type FTP struct {
 	Name              string     `mapstructure:"name"`
 	Address           string     `mapstructure:"address"`
 	Port              uint       `mapstructure:"port"`
-	Username          string     `mapstructure:"user"`
+	User              string     `mapstructure:"user"`
 	Password          string     `mapstructure:"password"`
+	PublicKey         string     `mapstructure:"public_key"`
 	Path              string     `mapstructure:"path"`
 	Period            uint       `mapstructure:"period"`
-	GzipLevel         uint8      `mapstructure:"gzip_level"`
+	GzipLevel         uint       `mapstructure:"gzip_level"`
```

Comment:
Hm. So `uint8` makes sense for this field.

* [S3 uses `uint`](https://github.com/fastly/go-fastly/blob/d5f2d007fd1adeb514b6efd8927913fff21be6b4/fastly/s3.go#L32)
* [GCS uses `uint8`](https://github.com/fastly/go-fastly/blob/d5f2d007fd1adeb514b6efd8927913fff21be6b4/fastly/gcs.go#L21)

Reply:
@phamann What's your thought on this?

Reply:
Unfortunately, we can't make this change because it would break backward compatibility.